ImmuDB4Net/Connection.cs

```diff
@@ -60,6 +61,34 @@ public async Task Shutdown()
         await Task.WhenAny(shutdownTask, Task.Delay(shutdownTimeout));
         channel = null;
     }
+
+    public void Shutdown()
+    {
+        if (channel == null)
+        {
+            return;
+        }
+        Task shutdownTask;
+        using (ManualResetEvent mre = new ManualResetEvent(false))
+        {
+            shutdownTask = Task.Factory.StartNew(async () =>
```

Comment:
I think I've seen somewhere a call to `task.Wait` which should wait for the task to be finished, is that correct? Can't we use that one instead?

Reply:
before you mentioned this I was sure that Task.Wait is awaitable. I'm glad that I was wrong and I've just fixed it as you suggested. It's simpler.